script.js

```diff
@@ -22,3 +24,248 @@ for (let i = 0; i < menuLinks.length; i += 1) {
+            type and scrambled it 1960s. 
+           </p>
+
+           <p>
+            Lorem Ipsum is simply dummy text of
+            the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a galley of type and scrambled it 1960s with
+            the releax map lapora verita.
+           </p>
+         </div>
+         
+         <div class="modal-btns">
+          <div class="see-live">
+            <a href="#">See Live  <img src="/assets/Icon-see live.svg" alt=""></a>
+          </div>
+          <div class="see-source">
+            <a href="#">See Source  <img src="/assets/Vector.png" alt=""></a>
+          </div>
+         </div>
+        </div>
+      </div>
+    </div>
+  </div>
+    `;
+    modalBackground.setAttribute('class', 'modal-background');
+    modalBackground.innerHTML = modalBoxContent;
+    workSection.appendChild(modalBackground);
+    document.body.style.overflowY = 'hidden';
+  }
+
+  if (e.target.classList.contains('btn-bg')) {
+    const modalBackground = document.createElement('div');
+    const modalBoxContent = `
+   <div class="modal-box">
+   <div class="modal-inner">
+     <div class="modal-title" >
+       <div class="modal-header">
+         <h3>Keeping track of hundreds of components website</h3>
+         <div>
+           <img src="./assets/Icon.png" class="close-menu" alt="close-menu" />
+         </div>
+       </div>
+       <br>
+       <ul class="modal-list">
+         <li class="tools-1">HTML</li>
+         <li class="tools-1">Boostrap</li>
+         <li class="tools-1">Ruby on rails</li>
+       </ul>
+     </div>
+     <div class="modal-img-box">
+       <div class="modal-img">
+         <img src="/assets/Snapshoot Portfolio.png" alt="" />
+       </div>
+
+       <div class="modal-details">
+        <div class="modal-info">
+         <p>
+           Lorem Ipsum is simply dummy text of the printing and typesetting
+           industry. Lorem Ipsum has been the industry's standard dummy text
+           ever since the 1500s, when an unknown printer took a galley of
+           type and scrambled it 1960s. 
+          </p>
+
+          <p>
+           Lorem Ipsum is simply dummy text of
+           the printing and typesetting industry. Lorem Ipsum has been the
+           industry's standard dummy text ever since the 1500s, when an
+           unknown printer took a galley of type and scrambled it 1960s with
+           the releax map lapora verita.
+          </p>
+        </div>
+        
+        <div class="modal-btns">
+         <div class="see-live">
+           <a href="#">See Live  <img src="/assets/Icon-see live.svg" alt=""></a>
+         </div>
+         <div class="see-source">
+           <a href="#">See Source  <img src="/assets/Vector.png" alt=""></a>
+         </div>
+        </div>
+       </div>
+     </div>
+   </div>
+ </div>
```

Comment:
:red_circle: The data inside your modal should be dynamic, and not static as it is now. The modal should fetch data depending on the selected card. Please make sure the data you display inside your modal-box is related to the card clicked.